instrumentation/kafka-streams/README.md

```diff
@@ -95,6 +95,25 @@ referencing the parent context stored on Headers, if available.
 Be aware that operations that require `builder.transformer(...)` will cause re-partitioning when
 grouping or joining downstream ([Kafka docs](https://kafka.apache.org/documentation/streams/developer-guide/dsl-api.html#applying-processors-and-transformers-processor-api-integration)).
 
+### Why not to trace _every_ Kafka Streams operation?
```

Comment:
I'd change it into "Should I trace _every_ Kafka Streams operation?", not a native but feels like more representing the intention of this note.

Reply:
"Why doesn't this trace all Kafka Streams operations?"